### DISCUSSION TITLE
大佬能打包 Kook 吗？没有Mac版本，但有 web 版。

### DISCUSSION THREAD

**Discussion by ghost** [tw93/Pake#84] [Ideas]
官方没有提供 Mac 版

**Comment by tw93** [comment]
请问是这个 https://www.kookapp.cn/  吗？ 试了试不太行，即使我改写user_agent欺骗他，也不行，这样的效果
![image](https://user-images.githubusercontent.com/8736212/203797791-c5cfc8d4-16d4-4b76-9bad-94f2a1de9284.png)

**Reply by ghost** [reply]
好的，还是非常感谢😘
